pulp_2_tests/tests/rpm/api_v2/test_copy.py

```diff
@@ -459,6 +487,47 @@ def copy_units(self, recursive, recursive_conservative, old_dependency):
         )
         return self.client.get(repos[1]['_href'], params={'details': True})
 
+    def copy_and_filter_units(self):
+        """
+        Copy units using ``recursive_conservative`` filtering OUT walrus-5.21.
+        (see Pulp issue #6151 for the reproducer this is based on)
+        """
+        repos = []
```

Comment:
I'd prefer the repos were explicitly labelled, I'm not sure I see the benefit of keeping a list of them?

Reply:
This is following the lead of the rest of this test-class (and the others in test_copy) - see copy_units(). Goal here is to test the single new issue, not to refactor the test infra :)

Reply:
Haha, fair :)